server/sonar-server/src/main/java/org/sonar/server/computation/dbcleaner/ProjectCleaner.java

```diff
@@ -68,19 +62,11 @@ public ProjectCleaner purge(DbSession session, IdUuidPair idUuidPair, Settings p
     cleanHistoricalData(session, configuration.rootProjectIdUuid().getId(), projectSettings);
     doPurge(session, configuration);
 
-    deleteIndexedIssuesBefore(idUuidPair.getUuid(), configuration.maxLiveDateOfClosedIssues());
-
     session.commit();
     logProfiling(start, projectSettings);
     return this;
   }
 
-  private void deleteIndexedIssuesBefore(String uuid, @Nullable Date lastDateWithClosedIssues) {
```

Comment:
For information, the goal of this PR is to be able to purge issues from the index whatever the value of the date is (because as we can see here when the date is null, nothing is done !)